zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -94,16 +111,59 @@ def __init__(self):
+    def decode_redundance_and_placement_policy(self, redundance_and_placement_policy):
+        if not redundance_and_placement_policy:
+            return None
+
+        try:
+            d = jsonobject.loads(base64.b64decode(redundance_and_placement_policy))
+            return self.RedundanceAndPlacementPolicy(
+                copyset_number=d.copysetNum,
+                replica_number=d.replicaNum,
+                zone_number=d.zoneNum
+            )
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)